### PR TITLE
[SDPA-1213] Upgraded JSON:API 2.0, JSON:API Extras 3.3, Webform 5.2, and Metatag.

### PR DESCRIPTION
**Pull request by sonnykt**
JIRA: https://digital-engagement.atlassian.net/browse/SDPA-1213

Related PRs:
* Tide Core: https://github.com/dpc-sdp/tide_core/pull/57
* Tide Media: https://github.com/dpc-sdp/tide_media/pull/19
* Tide API: https://github.com/dpc-sdp/tide_api/pull/28
* Tide Site: https://github.com/dpc-sdp/tide_site/pull/18
* Tide Webform: https://github.com/dpc-sdp/tide_webform/pull/21
* Tide News: https://github.com/dpc-sdp/tide_news/pull/14
* Tide Page: https://github.com/dpc-sdp/tide_page/pull/14
* Tide Event: https://github.com/dpc-sdp/tide_event/pull/24
* Tide Landing Page: https://github.com/dpc-sdp/tide_landing_page/pull/63
* Tide Monsido: https://github.com/dpc-sdp/tide_monsido/pull/13
* Tide Search: https://github.com/dpc-sdp/tide_search/pull/11
* Tide: https://github.com/dpc-sdp/tide/pull/39
* Content-VIC: https://github.com/dpc-sdp/content-vic-gov-au/pull/462

### CODE REVIEW

File: tide.info.yml [hunk 1]
```diff
@@ -4,16 +4,16 @@ type: profile
 package: Tide
 core: '8.x'
 dependencies:
-  - dpc-sdp:tide_api
   - dpc-sdp:tide_core
-  - dpc-sdp:tide_event
-  - dpc-sdp:tide_landing_page
+  - dpc-sdp:tide_api
```
Comment (kurtfoster) [inline tide.info.yml]: @sonnykt , any reason to move these out of alphabetical order?

Comment (sonnykt) [reply tide.info.yml]: @kurtfoster please refer to the TODO section in the ticket. There are several issues regarding to incorrect config placement and they cause issue when tide_landing_page is installed before tide_news. Changing the order is just an interim workaround for the tide_landinging_page module to be installed last. We need a proper fix for the config issues.

Comment (GROwen) [inline tide.info.yml]: @sonnykt Is there a reason to change the order of dependencies?

Asking to satisfy my OCD and love of alphanumeric ordering 😄

Comment (sonnykt) [reply tide.info.yml]: @GROwen please refer to my reply to @kurtfoster question above.

**Pull request merged**